packages/client/mutations/NavigateMeetingMutation.ts

```diff
@@ -142,6 +146,11 @@ export const navigateMeetingTeamUpdater: SharedUpdater<NavigateMeetingMutation_t
       reflectPrompt?.setValue([], 'editorIds')
     })
   }
+
+  const reflectionGroups = meeting.getLinkedRecords('reflectionGroups')
+  const sortedReflectionGroups = reflectionGroups.sort((a, b) =>
```

Comment:
`reflectionGroups.slice().sort`, sort mutates the current array instead of doing a shallow clone.
relay requires a new object when you call `setLinkedRecords` because internally it does shallow object equality comparisons